pkg/setup/setup.go

```diff
@@ -366,6 +366,11 @@ func (s *Setup) installPackageServicesPlugins(wg *sync.WaitGroup, httpClient *ht
 		return
 	}
 
+	if len(result.Packages) > 7 {
+		s.logger.Debug("More than 7 packages installed on the cluster, skipping the auto-installation of their subcommands")
```

Comment:
Why choosing 7 specifically? Can we make the debug message more generic? It might not be very informative for someone who is not aware that the operation takes a long time in some cases.